src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -126,7 +126,7 @@ def create(self, username, password, ttl: Optional[int] = None):
                             ]
                         }
                     }
-                Settings.MULTICLUSTER_CONFIG = multicluster_config
+                Settings.MULTICLUSTER_CONFIG = json.dumps(multicluster_config)
```

Comment:
Few months back i was trying to write a cleanup to the multi cluster api and then i removed this particular code in the auth file and i havent noticed any particular issue there. But i couldn't complete the PR because of other issues. In any case @aaSharma14 can confirm it better.

Reply:
We need to set the initial configs for local cluster when a user logs in the cluster such as cluster url, fsid, token and all, that's why its here

Reply:
@aaSharma14 that doesn't make any sense to me: I see that in my vstart/ceph-dev cluster without any multi-cluster configured, I have this mon KV entry configured... So every time someone logs in, this is overwritten and configured again, right?. What's the sense of it?

If this config setting requires initialization, it should be performed when the multi-cluster wizard is first run, but not by default... and not in an endpoint/controller that has nothing to do with multi-cluster.

Reply:
Confirmed: the first user that logs in will get its name there forever:
```
ceph config get mgr mgr/dashboard/MULTICLUSTER_CONFIG
{'current_url': 'https://localhost:11000', 'current_user': 'test', 'hub_url': 'https://localhost:11000', 'config': {'5444ea4b-fbb6-4c49-8eb0-ed12668baebc': [{'name': '5444ea4b-fbb6-4c49-8eb0-ed12668baebc', 'url': 'https://localhost:11000', 'cluster_alias': 'local-cluster', 'user': 'test'}]}}
```

What's the sense of it? I don't get it.

Reply:
Since, this not related and will be out of scope for this PR, let me do this cleanup in a separate PR.

Reply:
@epuertat , opened this new PR - https://github.com/ceph/ceph/pull/59666 for the cleanup. PTAL